### PR TITLE
Updates readme with nubook link

### PR DESCRIPTION
**Pull request by KipHamiltons**
The readme is now obsolete. NUbook has taken over.
Addresses #425 
[Preview](https://github.com/NUbots/NUbots/tree/hamiltons/clean_readme) ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,288 +4,7 @@ NUbots is a team in the University of Newcastle's robotics research group focuse
...
----
-
-**NOTE**
-Any changes you make outside of the `/home/nubots/NUbots` or `/home/nubots/build` directories will not persist after exiting the Docker shell.
-
----
-
-</details>
+[NUbook](https://nubook.nubots.net/) is the handbook and high-level documentation for the NUbots team.
+It contains guides for getting started and information for anyone interested in the team or any aspect of it.
```
Comment (JosephusPaye) [inline README.md:10]: Some direct links:

```suggestion
It contains guides for [getting started with development](https://nubook.nubots.net/guides/main/getting-started) and [information about the team](https://nubook.nubots.net/team/introduction) for anyone interested in any aspect of it.
```

**Review by JosephusPaye — CHANGES_REQUESTED** [review]
Looks good, just one change.

File: README.md [hunk 2]
```diff
@@ -4,288 +4,7 @@ NUbots is a team in the University of Newcastle's robotics research group focuse
...
----
-
-**NOTE**
-Any changes you make outside of the `/home/nubots/NUbots` or `/home/nubots/build` directories will not persist after exiting the Docker shell.
-
----
-
-</details>
+[NUbook](https://nubook.nubots.net/) is the handbook and high-level documentation for the NUbots team.
+It contains guides for [getting started with development](https://nubook.nubots.net/guides/main/getting-started) and [information about the team](https://nubook.nubots.net/team/introduction) for anyone interested in any aspect of it.
```
Comment (ysims) [inline README.md:10]:
```suggestion
It contains guides for [getting started with development](https://nubook.nubots.net/guides/main/getting-started) and [information about the team](https://nubook.nubots.net/team/introduction).
```
I didn't like the way it sounded. But feel free to dismiss.

Comment (KipHamiltons) [reply README.md:10]: up to you @JosephusPaye

Comment (JosephusPaye) [reply README.md:10]: Looks good, was trying to keep the general gist of your original @KipHamiltons.

Comment (KipHamiltons) [reply README.md:10]: Appreciate it! I am easy though. I'll commit this suggestion.

**Review by JosephusPaye — APPROVED** [review]
LGTM

**Pull request merged**